.github/workflows/flow-sanitizer.yml

```diff
@@ -0,0 +1,25 @@
+name: Clang Sanitizer
+
+# Documentation: https://redislabs.atlassian.net/wiki/spaces/DX/pages/3967844669/RediSearch+CI+refactor
+
+on:
+  workflow_call:
+
+jobs:
+  clang-sanitizer:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash -l -eo pipefail {0}
+    container:
+      image: "ubuntu:jammy"
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/run-tests
+        with:
+          container: "ubuntu:jammy"
```

Comment:
I wonder if we could just pass it like `${{ container.image }}` here instead of the string directly like that?